WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -7,12 +8,14 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets
 	public partial class WalletSettingsViewModel : RoutableViewModel
 	{
 		[AutoNotify] private bool _preferPsbtWorkflow;
+		[AutoNotify] private bool _autoCoinJoin;
 
 		public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 		{
 			var wallet = walletViewModelBase.Wallet;
 			Title = $"{wallet.WalletName} - Wallet Settings";
 			_preferPsbtWorkflow = wallet.KeyManager.PreferPsbtWorkflow;
+			_autoCoinJoin = Services.UiConfig.AutoCoinJoin;
```

Comment:
This should not be in `UiConfig`. All the settings that apply only to the wallet must be in `Wallet.KeyManager`.

Reply:
Done